nixos/modules/services/networking/xrdp.nix

```diff
@@ -44,7 +48,12 @@ in
 
       enable = mkEnableOption (lib.mdDoc "xrdp, the Remote Desktop Protocol server");
 
-      package = mkPackageOption pkgs "xrdp" { };
+      package = mkPackageOptionMD pkgs "xrdp" { };
+
+      audio = {
+        enable = mkEnableOption (lib.mdDoc "enable audio support for xrdp sessions. So far it only works with pulseaudio sessions on the server side. No pipewire.");
```

Comment:
Since `mkEnableOption` already adds `Whether to enable` at start and `.` at end:

```suggestion
        enable = mkEnableOption (lib.mdDoc "audio support for xrdp sessions. So far it only works with pulseaudio sessions on the server side. No pipewire");
```

Or overwrite the description by passing `// { description = ""; }`.

Reply:
I did as the code suggestion